x-pack/plugins/security_solution/cypress/integration/alerts_detection_exceptions.spec.ts

```diff
@@ -0,0 +1,230 @@
+      esArchiverUnload('auditbeat_for_exceptions3');
+      removeSignalsIndex();
+    });
+
+    it('Creates an exception and deletes it', () => {
+      loginAndWaitForPageWithoutDateRange(DETECTIONS_URL);
+      waitForAlertsIndexToBeCreated();
+      goToManageAlertsDetectionRules();
+      goToRuleDetails();
+
+      cy.get(RULE_STATUS).should('have.text', '—');
+
+      esArchiverLoad('auditbeat_for_exceptions');
+      activatesRule();
+      waitForTheRuleToBeExecuted();
+      waitForAlertsToPopulate();
+      deactivatesRule();
+      refreshPage();
+
+      cy.scrollTo('bottom');
+      cy.get(SERVER_SIDE_EVENT_COUNT)
+        .invoke('text')
+        .then((numberOfInitialAlertsText) => {
+          cy.wrap(parseInt(numberOfInitialAlertsText, 10)).should(
+            'eql',
+            NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS
+          );
+        });
+
+      goToExceptionsTab();
+      addsExceptionFromRuleSettings(exception);
+      activatesRule();
+
+      esArchiverLoad('auditbeat_for_exceptions2');
+      refreshPage();
+      waitForTheRuleToBeExecuted();
+      goToAlertsTab();
+
+      cy.scrollTo('bottom');
+      cy.get(SERVER_SIDE_EVENT_COUNT)
+        .invoke('text')
+        .then((numberOfAlertsAfterCreatingExceptionText) => {
+          cy.wrap(parseInt(numberOfAlertsAfterCreatingExceptionText, 10)).should('eql', 0);
+        });
+
+      goToClosedAlerts();
+      refreshPage();
+
+      cy.scrollTo('bottom');
+      cy.get(SERVER_SIDE_EVENT_COUNT)
+        .invoke('text')
+        .then((numberOfClosedAlertsAfterCreatingExceptionText) => {
+          cy.wrap(parseInt(numberOfClosedAlertsAfterCreatingExceptionText, 10)).should(
+            'eql',
+            NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS
+          );
+        });
+
+      goToOpenedAlerts();
+      refreshPage();
+
+      cy.scrollTo('bottom');
+      cy.get(SERVER_SIDE_EVENT_COUNT)
+        .invoke('text')
+        .then((numberOfOpenedAlertsAfterCreatingExceptionText) => {
+          cy.wrap(parseInt(numberOfOpenedAlertsAfterCreatingExceptionText, 10)).should('eql', 0);
+        });
+
+      goToExceptionsTab();
+      removeException();
+      esArchiverLoad('auditbeat_for_exceptions3');
+      refreshPage();
+      goToAlertsTab();
+      waitForTheRuleToBeExecuted();
+      waitForAlertsToPopulate();
+      refreshPage();
+
+      cy.scrollTo('bottom');
+      cy.get(SERVER_SIDE_EVENT_COUNT)
+        .invoke('text')
+        .then((numberOfAlertsAfterRemovingExceptionsText) => {
+          cy.wrap(parseInt(numberOfAlertsAfterRemovingExceptionsText, 10)).should(
+            'eql',
+            NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS
+          );
+        });
+    });
+  });
+
+  context('From alert', () => {
+    before(() => {
+      esArchiverLoad('rule_for_exceptions_from_alert');
+    });
+
+    after(() => {
+      esArchiverUnload('rule_for_exceptions_from_alert');
+      esArchiverUnload('auditbeat_for_exceptions_from_alert');
```

Comment:
You should be able to reuse the `auditbeat_for_exceptions` archive because now it's unloaded correctly here:

https://github.com/elastic/kibana/pull/81759/files#diff-b63f9bf0d777ee3906226b1ba66f112bff14ded727a9b95b7dcfb54cece691cfR47

Previously these weren't being unloaded correctly, which is why I think we needed to duplicate them.